appveyor.yml

```diff
@@ -12,7 +12,11 @@ environment:
     - RUNTIME: '3.6'
       TOOLKITS: "null"
     - RUNTIME: '3.6'
-      TOOLKITS: "pyside2 pyqt"
+      TOOLKITS: "pyqt"
+    - RUNTIME: '3.6'
+      TOOLKITS: "pyqt5"
+    - RUNTIME: '3.6'
+      TOOLKITS: "pyside2"
```

Comment:
any specific reason you chose to split them up into multiple jobs instead of running tests for all toolkits in the same job?

Reply:
Yes... because I "messed up" the `appveyor-run.cmd` in https://github.com/enthought/traitsui/pull/940
The old `appveyor-run.cmd` took me a while to understand and I misunderstood it.  If I did not split it up, the current `appveyor-run.cmd` will only run test for the first toolkit in the space separated string.
I am not keen on reverting the `appveyor-run.cmd` just so that we can lump all these into one build. It is at best an optimization for maybe almost 2 mins of build time.  Splitting them up is more readable, with the benefit that now we can run tests on wx even if the test fails for an earlier toolkit.

Reply:
fine by me